### ISSUE TITLE
Keyboard shortcuts not working!!!

### ISSUE THREAD

**Issue by hsiayuchih** [Genymobile/scrcpy#2443]
I am not being able to use any of the ctrl+___ shortcuts when using Scrcpy. I am using it on windows10 21h1. I have tried to close the input method, but also!

**Comment by rom1v** [comment]
Why scrcpy version?
Which shortcut modifier key do you use? (ctrl, alt, …?)

**Comment by rldmt** [comment]
On Ubuntu 20.4 , scrcpy version v1.12.1 these shortcuts do not work:
Ctrl+p - turn screen on/off
Ctrl+r - rotate device screen
Ctrl+shift+n - collapse notifivation panel
Ctrl+c, ctrl+v - copy paste

**Comment by rom1v** [comment]
> On Ubuntu 20.4 , scrcpy version v1.12.1 these shortcuts do not work:

That's unexpected. Are these the only shortcuts which do not work for you? (Are the other shortcuts working?)

**Comment by hsiayuchih** [comment]
> 哪个 scrcpy 版本？
> 您使用哪个快捷键修饰键？（ctrl，alt，……？）

I have tried 1.17 and 1.18 but none of them can be worked.
ctrl

**Comment by rom1v** [comment]
With v1.17 or v1.18, the default shortcut mod is Alt.

https://github.com/Genymobile/scrcpy#shortcuts

**Comment by edmundlaugasson** [comment]
Some shortcut keys interfere with desktop environment ones, like Super + arrow keys to rotate display. But CTRL+R works.
CTRL+SHIFT+O didn't work for me to turn device screen back on. But CTRL+O to turn device screen off, worked and it is really useful.

CTRL+G is also useful, but resizing back is quite tricky for now. E.g. rotating YouTube video will return initial size. Fit window shortcut key would be useful. Also moving window, resizing it might help.

Copying from computer to phone succeeded easily, but not vice versa. Somehow I succeeded, but couldn't repeat that. Sometimes quickly twice CTRL+C helped, sometimes not. Using Android own copy didn't help. Sometimes it seems, that copying happens with huge delay.

Using Linux Mint 20.2 Cinnamon, other relevant software:
```
scrcpy -v
scrcpy 1.12.1

dependencies:
 - SDL 2.0.10
 - libavcodec 58.54.100
 - libavformat 58.29.100
 - libavutil 56.31.100
```
Phone has Android 9.0.6 (OxygenOS at Oneplus).
Unfortunately Ubuntu and its based Linux distros have sometimes quite outdated software packages.

**Comment by rom1v** [comment]
> scrcpy 1.12.1

> Some shortcut keys interfere with desktop environment ones, like Super + arrow keys to rotate display.

It's not that they interfere, it's just that the feature has been implemented in v1.13. Anyway, you can customize the shortcut modifier: https://github.com/Genymobile/scrcpy#shortcuts

> Unfortunately Ubuntu and its based Linux distros have sometimes quite outdated software packages.

You can install the latest version easily: https://github.com/Genymobile/scrcpy/blob/master/BUILD.md#simple

**Comment by edmundlaugasson** [comment]
That's a good point to change mod key.
Anyway, installing software manually is a bad idea. There will be issues with dependencies, later updating might be challenging. The proper way is to make own repository and prepare appropriate packages to particular Linux distro by own. That in turn might be overwhelming for most of us. That's, why distro makers carefully look into compatibility issues and update a distro as a whole and not just particular package. I guess there should be either repository, we can add and install newer version or choose such Linux distro, that has all needed features and that particular software up-to-date, in current case the scrcpy package. As Ubuntu and it's based Linux Mint has many users, then it is certainly a big interest group. Although I consider Manjaro Linux as an alternative to Linux Mint, this is not so easy change whole system and solve all challenges. Manjaro and it's basis Arch Linux are tend to use latest and greatest software, having rolling release model, that is way more sustainable, than reinstalling system all the time, when new major release is out.
